### ISSUE TITLE
Allow specifying device name and metadata in the tools app 

### ISSUE THREAD

**Issue by zmarkan** [getditto/DittoAndroidTools#136]
Tools app should let you pass device names to each device so that we can more easily identify them with presence viewer or other mesh observability tools.
Right now, Ditto defaults to device model. 

See below image with all devices of the same type in the presence viewer:

![IMG_4581](https://github.com/user-attachments/assets/377f9b53-d20f-4996-bba2-867b6caf369a)

**Comment by bplattenburg** [comment]
Whoever picks this up - check if we need the same in Swift and create a matching issue if so

**Comment by zmarkan** [comment]
Yes - swift tools app just names them like iPad and iPhone - not even taking the device name from iOS into account.

**Comment by kndoshn** [comment]
Please note that if you change the `deviceName`, you need to restart the Ditto sync to update the name:
> Changes to this property after startSync() was called will only take effect after the next restart of sync.
https://software.ditto.live/android/Ditto/4.8.0/api-reference/ditto/live.ditto/-ditto-base/device-name.html

**Comment by zmarkan** [comment]
In Richard's implementation on iOS there's already logic for this as you configure various sync mechanisms - I think we can use that screen for general config of the tools app
